File: curso_java/Switch_case.java
import java.util.Scanner;
public class Switch_case {
    public static void main(String[] args) {
        int dia_semana;
        Scanner s;
        s = new Scanner(System.in);

        System.out.print("Dia da semana: ");
        dia_semana = s.nextInt();

        switch(dia_semana) {
        case 1: 
        System.out.println("Domingo"); 
        break;
        case 2: 
        System.out.println("Segunda"); 
        break;
        case 3: 
        System.out.println("Terça");
        break;
        case 4:
        System.out.println("Quarta"); 
        break;
        case 5: 
        System.out.println("Quinta"); 
        break;
        case 6: 
        System.out.println("Sexta"); 
        break;
        case 7: 
        System.out.println("Sábado"); 
        break;
        default:
        System.out.println("O número digitado não corresponde a um dia da semana");

        }
    }
}

File: Curso_C/Aulas/Switch_case.c
#include <stdio.h>

/*
O algoritmo irá disponibilizar
uma opção a ser digitada para
mostrar o dia de acordo com
o número digitado pelo usuário
*/

int main() {
    int dia_semana;

    printf("Digite o número desejado: ");
    scanf("%i", &dia_semana);
 
    switch(dia_semana) {
        case 1:
        printf("Domingo\n");
        break;//quebra de ciclo
        case 2:
        printf("Segunda-feira\n");
        break;//quebra de ciclo
        case 3:
        printf("Terça-feira\n");
        break;//quebra de ciclo
        case 4:
        printf("Quarta-feira\n");
        break;//quebra de ciclo
        case 5:
        printf("Quinta-feira\n");
        break;//quebra de ciclo
        case 6:
        printf("Sexta-feira\n");
        break;//quebra de cico
        case 7:
        printf("Sábado\n");
        break;//quebra de ciclo
    }
}

    
